.env

```diff
@@ -0,0 +1,3 @@
+MONGO_HOST=uwtimeline-rvzny.mongodb.net/test?retryWrites=true&w=majority
+MONGO_USERNAME=Stephen_New
+MONGO_PASSWORD=vVQux86R0pzVcLtA
```

Comment:
should we be pushing this to github

Reply:
Shit

Reply:
https://help.github.com/en/articles/removing-sensitive-data-from-a-repository

Reply:
yeah forgot to include it in gitignore